docs/application-development/testing.md

```diff
@@ -111,7 +113,23 @@ Content is currently unavailable.
 </TabItem>
 <TabItem value="python">
 
-Content is currently unavailable.
+To test a Heartbeat in an Activity, you can use the `on_heartbeat()` property of the `ActivityEnvironment` class. This property allows you to set a custom function that will be called every time the `activity.heartbeat()` function is called within the Activity.
```

Comment:
```suggestion
To test a Heartbeat in an Activity, use the `on_heartbeat()` property in the `ActivityEnvironment` class to set a custom function. This function will be called every time the `activity.heartbeat()` function is called within the Activity.
```

Reply:
Just realized this was the generated doc. Please apply these changes to the source docs if desired.